opm/simulators/flow/FIBlackoilModel.hpp

```diff
@@ -59,23 +59,39 @@ class FIBlackOilModel : public BlackOilModel<TypeTag>
     FIBlackOilModel(Simulator& simulator)
         : BlackOilModel<TypeTag>(simulator)
     {
+#ifdef _OPENMP
+        const auto& gv = this->gridView_;
+        const auto num_elements = gv.size(0);
+        const int nt = omp_get_num_threads();
+        constexpr int max_chunk_size = 1000;
+        const int chunk_size = std::min(max_chunk_size, num_elements / nt);
+        OpmLog::debug(fmt::format("Using chunk size {} for property evaluation with {} OpenMP threads.", chunk_size, nt));
```

Comment:
Tiny nit: We should try to avoid using `fmt::format()` in a header since it means that any downstream or out-of-tree users of the header then have to have libfmt installed in a central location in order to consume the header.  In this case, maybe we could just use `std::to_string()` on the `chunk_size` and `nt` objects?

Reply:
Will fix